controllers/v1/criteriaController.js

```diff
@@ -65,127 +67,17 @@ module.exports = class Criteria extends Abstract {
           });
         }());
 
+        let updatedCriteria = await criteriasHelper.upload(
```

Comment:
@aman-tunerlabs - Can we rename it to criteriaHelper just to maintain consistency in naming helpers with module name. You might have to replace all across